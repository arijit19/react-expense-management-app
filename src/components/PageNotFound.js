import React from 'react';
import {Link} from 'react-router-dom';

const PageNotFound = ()=>(
    <p>404!!<Link to='/'> GO Home </Link></p>
);

export default PageNotFound;