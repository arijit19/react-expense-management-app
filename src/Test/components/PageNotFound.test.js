import React from 'react';
import {shallow} from 'enzyme';

import PageNotFound from '../../components/PageNotFound.js'

test('Testing Not found page component', ()=>{
    const wrapper = shallow(<PageNotFound/>);
    expect(wrapper).toMatchSnapshot();
})