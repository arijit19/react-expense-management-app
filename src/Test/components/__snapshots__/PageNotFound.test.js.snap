// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Not found page component 1`] = `
<p>
  404!!
  <Link
    to="/"
  >
     GO Home 
  </Link>
</p>
`;
